applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/harmonic_analysis_strategy.hpp

```diff
@@ -263,19 +263,20 @@ class HarmonicAnalysisStrategy
             {
                 for( auto& node : r_model_part.Nodes() )
                 {
-                    ModelPart::NodeType::DofsContainerType node_dofs = node.GetDofs();
+                    ModelPart::NodeType::DofsContainerType& node_dofs = node.GetDofs();
                     const std::size_t n_node_dofs = node_dofs.size();
                     const Matrix& r_node_eigenvectors = node.GetValue(EIGENVECTOR_MATRIX);
 
-                    if( node_dofs.IsSorted() == false )
-                    {
-                        node_dofs.Sort();
-                    }
+                    // TO BE VERIFIED!! In the current implmentation of Dofs there are nor reordered and only pushec back. 
+                    // if( node_dofs.IsSorted() == false )
+                    // {
+                    //     node_dofs.Sort();
+                    // }
```

Comment:
@qaumann please have a look, but I guess this comes from the EigensolverStrategy